packages/suite/package.json

```diff
@@ -27,6 +27,7 @@
         "@sentry/core": "^7.100.1",
         "@solana/buffer-layout": "^4.0.1",
         "@solana/web3.js": "^2.0.0",
+        "@solana/web3.js-version1": "npm:@solana/web3.js@1.95.8",
```

Comment:
this looks like a todo

Reply:
yea, Everstake has to migrate to v2 https://satoshilabs.slack.com/archives/C0543DJBK0C/p1733907109990759